gkeys/gkeys/actions.py

```diff
@@ -499,6 +509,7 @@ def speccheck(self, args):
                 'Expiry.................: %d' % len(set(failed['expired'])),
                 'Expiry Warnings........: %d' % len(set(failed['warn'])),
                 'SPEC requirements......: %d' % len(set(failed['spec'])),
+                'Incorrect qualified IDs: %d' % len(set(failed['qualified_id'])),
```

Comment:
I think using "Incorrect" here is not right.

Since the id's that a key may contain be perfectly valid, yet not meet the specification requirement.
And since it is displayed under the Found Failures section. It should just be "Qualified ID...: %d"

            ['\nFound Failures:\n-------',
                'Revoked................: %d' % len(set(failed['revoked'])),
                'Invalid................: %d' % len(set(failed['invalid'])),
                'No Signing subkey......: %d' % len(set(failed['sign'])),
                'No Encryption subkey...: %d' % len(set(failed['encrypt'])),
                'Algorithm..............: %d' % len(set(failed['algo'])),
                'Bit length.............: %d' % len(set(failed['bits'])),
                'Expiry.................: %d' % len(set(failed['expired'])),
                'Expiry Warnings........: %d' % len(set(failed['warn'])),
                'SPEC requirements......: %d' % len(set(failed['spec'])),
                'Incorrect qualified IDs: %d' % len(set(failed['qualified_id'])),
                '=============================',
                'SPEC Approved..........: %d' % len(set(failed['spec-approved'])),
            ])

Also, it should be above the SPEC requirements subtotal.